src/features/werkbericht/service.ts

```diff
@@ -142,24 +146,30 @@ export function useWerkberichten(
     if (typesResult.state !== "success" || skillsResult.state !== "success")
       return "";
 
-    const url = window.gatewayBaseUri + "/api/openpub/kiss_openpub_pub";
-    if (!parameters?.value) return url;
+    if (!parameters?.value) return BERICHTEN_BASE_URI;
 
     const { typeId, search, page, skillIds } = parameters.value;
-    const params: [string, string][] = [["orderby", "modified"]];
```

Comment:
This was implemented, but I was told by the backend team that `?orderby=modified` does not work and was therefore redundant.